pkg/executor/sortexec/sort_spill.go

```diff
@@ -101,7 +101,13 @@ func (s *parallelSortSpillAction) Action(t *memory.Tracker) {
 	}
 
 	if t.CheckExceed() && !hasEnoughDataToSpill(s.spillHelper.sortExec.memTracker, t) {
-		s.GetFallback().Action(t)
+		s.triggerFallBackAction(t)
+	}
+}
+
+func (s *parallelSortSpillAction) triggerFallBackAction(tracker *memory.Tracker) {
```

Comment:
can we define this `triggerFallBackAction` in the base action, so there is no need to define it in both `parallelSortSpillAction` and `topNSpillAction`

Reply:
> can we define this `triggerFallBackAction` in the base action, so there is no need to define it in both `parallelSortSpillAction` and `topNSpillAction`

done